source/concepts/rewards.rst

```diff
@@ -0,0 +1,107 @@
+###############
+Reward Programs
+###############
+
+:doc:`Nodes <node>` **are the backbone of any blockchain**, since they are the ones creating and storing the blocks, among other things. It is therefore **critical** that as many nodes as possible are active in the network at all times.
+
+Beyond the direct **block rewards** (:doc:`fees` and :doc:`inflation`) that nodes obtain for producing blocks, |codename| **provides a series of special Reward Programs** aimed at further encouraging users to create and maintain nodes.
+
+Each of these programs rewards **a different aspect of the network** by distributing |networkcurrency| among all program participants.
+
+Node owners whishing to benefit from these programs must explicitly **enroll** to them. The descriptions below contain **guides** explaining how to enroll to each program.
+
+*****************
+Supernode Program
+*****************
+
+This program aims at ensuring that a sufficiently large number of **powerful nodes** will always be available to guarantee the smooth functioning of the network.
+
+Requirements
+============
+
+Supernodes are regular nodes meeting the following additional hardware and operational requirements:
+
+.. csv-table::
+    :header: "Parameter", "Requirement"
+    :delim: ;
+    :widths: 30 70
+
+    **Node type**; Must be a Dual node (Both a :ref:`Peer <peer-node>` and an :ref:`API <api-node>` node).
```

Comment:
A supernode must also include a rest gateway

Reply:
How does one include that? I was under the impression API nodes already provided REST access... What is exactly a REST gateway **from a user's perspective**?

Reply:
"The node must include a Rest Gateway"

The "full" dual node has 4 components:
- Server node in peer and api mode. Port 7900
- Broker that keeps the mongo DB up to date and provides the WebSockets queues. Port 7902
- Mongo db
- Rest Gateway (aka catapult-rest) that reads data from the mongo db. Port 3000. This the facade for regular apps like explorer, cli, wallet, etc.

Our api/dual presets on bootstrap add rest gateways by default. But if you are manually creating a supernode, you need to install the javascript catapult-rest.

The theory goes, the user could have a dual node without a rest gateway. Supernode program requires a rest gateway so services like wallet, explore could potentially talk to the node. The node monitor also uses it to validate the state of the node (like height and server version).

Reply:
I've added a note, and will include the information regarding the REST Gateway once I write the guide on setting up a node that includes a REST Gateway :)